### DISCUSSION TITLE
Nehirden kayıkla karşıya geçme sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#278] [Zeka Sorusu]
Bir nehrin bir yakasında 4 evli çift bulunmaktadır.
Sahip oldukları tek kayığı kullanarak karşıya geçmek istemektedirler.
Şartlar şunlardır:

Kayığa en fazla iki kişi binebilir.
Nehrin ortasında, iki kıyı arasında, istedikleri zaman durabilecekleri bir ada bulunmaktadır.
Hiçbir kadın, yanında kocası yokken başka bir adamla kıyıda, adada veya kayıkta, bir anlığına da olsa beraber duramaz.( Adada iki yalnızca iki kadın varsa, kadınlardan birinin kocası adaya gelip eşini alamaz, çünkü bir anlığına da olsa diğer kadınla, eşi yanında değilken bir araya gelmiş olur.)
Karşıya nasıl geçerler?

**Comment by slos8888** [comment]
İlk adımda bir erkek ve bir kadın, yani bir çift, kayıkla adaya geçer. Erkek adadan geri döner, kayık diğer kıyıya gidip orada bırakılır.

Ardından başka bir erkek ve kadın çifti, yani ikinci çift, kayığa biner ve adaya geçerler. Kadın adadan geri döner ve erkek diğer kıyıya gidip orada kayığı bekler.

Üçüncü çift, kayığa biner ve adaya geçer. Erkek adadan döner ve kadın diğer kıyıya gidip orada kayığı bekler.

Son olarak, dördüncü çift, kayığa biner ve adaya geçer. Kadın adadan döner ve erkek diğer kıyıya gidip orada kayığı alır.

Bu şekilde, tüm evli çiftler nehrin diğer yakasına geçmiş olur ve tüm çiftler karşıya ulaşır.
